msteams-platform/bots/how-to/conversations/send-proactive-messages.md

```diff
@@ -15,7 +15,7 @@ A proactive message is any message sent by a bot that is not in direct response
 * Notifications
 * Scheduled messages
 
-In order for your bot to send a proactive message, it must have access to the user, group chat, or team that you wish to send the message to. For a group chat or team, this means the app that contains your bot must first be installed to that location. You can [proactively install your app using Graph](#proactively-install-your-app-using-graph) in a team if necessary, or use an [app policy](/microsoftteams/teams-custom-app-policies-and-settings) to push apps out to teams and users in your tenant. For users, your app either needs to be installed for that user, or your user needs to be part of a team where your app is installed.
+For your bot to send a proactive message, it must have access to the user, group chat, or team that you wish to send the message to. For a group chat or team, this means the app that contains your bot must be installed to that location first. You can [proactively install your app using Graph](#proactively-install-your-app-using-graph) in a team if necessary or use an [app policy](/microsoftteams/teams-custom-app-policies-and-settings) to push apps out to teams and users in your tenant. For users, your app either needs to be installed for that user or your user needs to be part of a team where your app is installed.
```

Comment:
The following sentence is not clear. 
For users, your app either needs to be installed for that user or your user needs to be part of a team where your app is installed.
Should it be, For the users to use the app, it must either be installed for the user or the user must be part of a team where your app is installed.

```Suggestion
For your bot to send a proactive message, it must have access to the user, group chat, or team that you want to send the message to. For a group chat or team, this means the app that contains your bot must be installed to that location first. You can [proactively install your app using Graph](#proactively-install-your-app-using-graph) in a team if necessary or use an [app policy](/microsoftteams/teams-custom-app-policies-and-settings) to push apps out to teams and users in your tenant. For users, your app either must be installed for that user or your user must be part of a team where your app is installed.
```